packages/core/strapi/package.json

```diff
@@ -181,10 +176,6 @@
   },
   "devDependencies": {
     "@strapi/ts-zen": "^0.2.0",
-    "@types/bcryptjs": "2.4.3",
-    "@types/configstore": "5.0.1",
-    "@types/delegates": "1.0.0",
-    "@types/find-root": "1.1.4",
```

Comment:
were configstore, delegate and bcrypt just forgotten when they were removed?

Reply:
we moved most of the code into the core package and they didn't get cleaned up :)